WiFi is unavailable at first time boot

This issue can be reproduced on image downloaded from this action: 
https://github.com/ubopod/ubo_app/actions/runs/13606591776/job/38038859598

Steps to reproduce: connect to WiFi hotspot after the first time boot-up. Enter WiFi credentials in Web UI and press provide. You would see the error below. If you try for the second time, you won't see this issue. I believe this might be a race condition as we attempt to enable WiFi (`nmcli radio wifi on`). I think we should attempt to run this command before `add_wireless_connection` in WiFi manager. 


```
1740866807.315225 [ERROR] wifi connection input - error while creating connection
Traceback (most recent call last):
  File "/opt/ubo/env/lib/python3.11/site-packages/ubo_app/services/030-wifi/pages/create_wireless_connection.py", line 165, in input_wifi_connection
    await add_wireless_connection(
  File "/opt/ubo/env/lib/python3.11/site-packages/ubo_app/services/030-wifi/wifi_manager.py", line 290, in add_wireless_connection
    connection = await wait_for(
                 ^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/asyncio/tasks.py", line 479, in wait_for
    return fut.result()
           ^^^^^^^^^^^^
  File "/opt/ubo/env/lib/python3.11/site-packages/sdbus/dbus_proxy_async_method.py", line 107, in _dbus_async_call
    reply_message = await bus.call_async(call_message)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sdbus_async.networkmanager.exceptions.NetworkManagerUnknownConnectionError: Connection 'CircuitLaunch' is not available on device wlan0 because device is not available
```

I also see the following error in the logs as the WiFi service is being initialized:

```
1740866714.995789 [ERROR] Event loop exception
Traceback (most recent call last):
  File "/opt/ubo/env/lib/python3.11/site-packages/debouncer/debounce.py", line 124, in act
    await _timer_expired(state)
  File "/opt/ubo/env/lib/python3.11/site-packages/debouncer/debounce.py", line 119, in _timer_expired
    await _trailing_edge(state, current_time)
  File "/opt/ubo/env/lib/python3.11/site-packages/debouncer/debounce.py", line 159, in _trailing_edge
    return await _invoke_func(state, time)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/ubo/env/lib/python3.11/site-packages/debouncer/debounce.py", line 143, in _invoke_func
    result = await result
             ^^^^^^^^^^^^
  File "/opt/ubo/env/lib/python3.11/site-packages/ubo_app/services/030-wifi/wifi_manager.py", line 108, in request_scan
    await wait_for(wifi_device.request_scan({}))
  File "/usr/lib/python3.11/asyncio/tasks.py", line 479, in wait_for
    return fut.result()
           ^^^^^^^^^^^^
  File "/opt/ubo/env/lib/python3.11/site-packages/sdbus/dbus_proxy_async_method.py", line 107, in _dbus_async_call
    reply_message = await bus.call_async(call_message)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sdbus_async.networkmanager.exceptions.NmDeviceNotAllowedError: Scanning not allowed while unavailable - extra: {
  "error_message": "Task exception was never retrieved",
  "future": "<Task finished name='Task-66' coro=<_timer_expired.<locals>.act() done, defined at /opt/ubo/env/lib/python3.11/site-packages/debouncer/debounce.py:122> exception=NmDeviceNotAllowedError('Scanning not allowed while unavailable')>",
  "loop": "<_UnixSelectorEventLoop running=True closed=False debug=False>",
  "parent_stack": "None"
}
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -73,6 +73,7 @@
 - fix(audio): make audio run the `unbind` and `bind` sequence if the audio card is not listed by `alsaaudio.cards`
 - fix(audio): replace wareshare repo with ubopod fork
 - refactor(wifi): show wifi onboarding only on ubo-pod, and initiate the wifi input over hotspot on non-ubo-pods
+- fix(audio): make sure `set_playback_mute`, `set_playback_volume` and `set_capture_volume` are applied on the audio card, even if audio manager is not initialized when these methods are called
 
 ## Version 1.2.2
 

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,6 +1,6 @@
 # Changelog
 
-## Upcoming
+## Version 1.3.0
 
 - fix: remove dependencies of `publish` job of `publish_to_pypi` workflow
 - fix: vscode binary now needs to be instructed about the location of the binary with `version use` sub-command - closes #217

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -72,6 +72,7 @@
 - refactor(services): avoid silencing caught exceptions in services by either reraising them so that they are caught and reported by the global exception handler or directly reporting them
 - fix(audio): make audio run the `unbind` and `bind` sequence if the audio card is not listed by `alsaaudio.cards`
 - fix(audio): replace wareshare repo with ubopod fork
+- refactor(wifi): show wifi onboarding only on ubo-pod, and initiate the wifi input over hotspot on non-ubo-pods
 
 ## Version 1.2.2
 

--- a/ubo_app/services/000-audio/audio_manager.py
+++ b/ubo_app/services/000-audio/audio_manager.py
@@ -44,6 +44,10 @@ class AudioManager:
         self.has_speakers = False
         self.has_microphones = False
 
+        self.playback_mute = True
+        self.playback_volume = 0.1
+        self.capture_volume = 0.1
+
         async def initialize_audio() -> None:
             for _ in range(TRIALS):
                 try:
@@ -55,6 +59,10 @@ class AudioManager:
                     logger.exception('No audio card found')
                     await send_command('audio', 'failure_report', has_output=True)
                 else:
+                    # In case they were set before the card was initialized
+                    self.set_playback_mute(mute=self.playback_mute)
+                    self.set_playback_volume(self.playback_volume)
+                    self.set_capture_volume(self.capture_volume)
                     break
                 await asyncio.sleep(1)
 
@@ -167,6 +175,7 @@ class AudioManager:
             Mute to set
 
         """
+        self.playback_mute = mute
         try:
             # Assume pulseaudio is installed
             mixer = alsaaudio.Mixer(control='Master')
@@ -196,6 +205,7 @@ class AudioManager:
             Volume to set, a float between 0 and 1
 
         """
+        self.playback_volume = volume
         if volume < 0 or volume > 1:
             msg = 'Volume must be between 0 and 1'
             raise ValueError(msg)
@@ -230,6 +240,7 @@ class AudioManager:
             Volume to set, a float between 0 and 1
 
         """
+        self.capture_volume = volume
         if volume < 0 or volume > 1:
             msg = 'Volume must be between 0 and 1'
             raise ValueError(msg)

--- a/ubo_app/services/030-wifi/setup.py
+++ b/ubo_app/services/030-wifi/setup.py
@@ -6,7 +6,6 @@ from typing import TYPE_CHECKING
 
 from debouncer import DebounceOptions, debounce
 from pages import create_wireless_connection, main
-from redux import AutorunOptions
 from wifi_manager import (
     get_connections,
     get_wifi_device,
@@ -38,6 +37,7 @@ from ubo_app.store.services.wifi import (
     WiFiUpdateRequestEvent,
 )
 from ubo_app.utils.async_ import create_task
+from ubo_app.utils.eeprom import get_eeprom_data
 from ubo_app.utils.network import get_saved_wifi_ssids, has_gateway
 from ubo_app.utils.persistent_store import (
     read_from_persistent_store,
@@ -121,8 +121,21 @@ async def _check_connection() -> None:
         },
     )
     if not await has_gateway() and not await get_saved_wifi_ssids():
-        logger.info('No network connection found, prompting for Wi-Fi input')
-        store.dispatch(WiFiInputConnectionAction())
+        if get_eeprom_data() is not None:
+            if not read_from_persistent_store(
+                key='wifi_has_visited_onboarding',
+                default=False,
+            ):
+                logger.info('No network connection found, showing WiFi onboarding.')
+                store.dispatch(
+                    NotificationsAddAction(
+                        notification=ONBOARDING_NOTIFICATION,
+                    ),
+                    WiFiSetHasVisitedOnboardingAction(has_visited_onboarding=True),
+                )
+        else:
+            logger.info('No network connection found, prompting for Wi-Fi input.')
+            store.dispatch(WiFiInputConnectionAction())
 
 
 def init_service() -> Subscriptions:
@@ -142,34 +155,12 @@ def init_service() -> Subscriptions:
         ),
     )
 
-    @store.autorun(
-        lambda state: state.ip.is_connected,
-        options=AutorunOptions(default_value=None),
-    )
-    def check_onboarding(is_connected: bool | None) -> None:
-        if is_connected is False and not read_from_persistent_store(
-            key='wifi_has_visited_onboarding',
-            default=False,
-        ):
-            logger.info('No internet connection, showing WiFi onboarding.')
-            store.dispatch(
-                NotificationsAddAction(
-                    notification=ONBOARDING_NOTIFICATION,
-                ),
-                WiFiSetHasVisitedOnboardingAction(has_visited_onboarding=True),
-            )
+    create_task(_check_connection())
 
-        if is_connected is not None:
-            check_onboarding.unsubscribe()
-
-    subscriptions = [
+    return [
         store.subscribe_event(WiFiUpdateRequestEvent, request_scan),
         store.subscribe_event(
             WiFiInputConnectionEvent,
             create_wireless_connection.input_wifi_connection,
         ),
     ]
-
-    create_task(_check_connection())
-
-    return subscriptions

